packages/token-search-discovery-controller/CHANGELOG.md

```diff
@@ -7,19 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+
 ### Added
 
-- Introduce the `logoUrl` property to the `TokenSearchApiService` response
+- Rename `ControllerMessenger` to `Messenger` ([#5243](https://github.com/MetaMask/core/pull/5243))
+- Bump @metamask/utils to v11.1.0 ([#5223](https://github.com/MetaMask/core/pull/5223))
+- Introduce the `logoUrl` property to the `TokenSearchApiService` response ([#5195](https://github.com/MetaMask/core/pull/5195))
   - Specifically in the `TokenSearchResponseItem` type
-- Introduce `TokenDiscoveryApiService` to keep discovery and search responsibilities separate
+- Introduce `TokenDiscoveryApiService` to keep discovery and search responsibilities separate ([#5214](https://github.com/MetaMask/core/pull/5214))
   - This service is responsible for fetching discover related data
   - Add `getTrendingTokens` method to fetch trending tokens by chain
   - Add `TokenTrendingResponseItem` type for trending token responses
 - Export `TokenSearchResponseItem` type from the package index
 
 ### Changed
 
-- Update the TokenSearchApiService to use the updated URL for `searchTokens`
+- Update the TokenSearchApiService to use the updated URL for `searchTokens` ([#5195](https://github.com/MetaMask/core/pull/5195))
   - The URL is now `/tokens-search` instead of `/tokens-search/name`
 - Changed the "name" parameter to "query" in the `searchTokens` method
```

Comment:
I see this entry below:

```
Changed the "name" parameter to "query" in the `searchTokens` method
```

Since consumers now have to adjust their code in response to this change, should we mark it as breaking? Additionally should we release 2.0.0 instead of 1.1.0?

``` suggestion
- **BREAKING:** The `searchTokens` method now takes a `query` parameter instead of `name`
- Update the TokenSearchApiService to use the updated URL for `searchTokens`
- Update the TokenSearchApiService to use the updated URL for `searchTokens` ([#5195](https://github.com/MetaMask/core/pull/5195))
  - The URL is now `/tokens-search` instead of `/tokens-search/name`
```

Reply:
Yes good point, we don't technically have any consumers but let's be I still agree with this change.  I'll update it.